starport/services/network/publish.go

```diff
@@ -104,6 +104,19 @@ func (n Network) Publish(ctx context.Context, c Chain, options ...PublishOption)
 		if err != nil {
 			return 0, 0, 0, err
 		}
+
+		n.ev.Send(events.New(events.StatusOngoing, "Fetching custom Genesis from URL"))
+		isTarball := false
+		genesisFile, isTarball, err = cosmosutil.GenesisFromTarball(genesisFile)
+		if err != nil {
+			return 0, 0, 0, err
+		}
+		if isTarball {
+			n.ev.Send(events.New(events.StatusDone, "Custom Genesis from URL tarball extracted"))
```

Comment:
```suggestion
			n.ev.Send(events.New(events.StatusDone, fmt.Sprintf("Custom Genesis from URL tarball extracted at path %s", path)))
```

Let's also show at which path we extracted it from